### PR TITLE
Peer Modal: add dark mode

### PR DESCRIPTION
**Pull request by ruibrii**
## Description

Add dark mode to the peer modal.

image
image

## Change Log

- [x] Make background for modal and cards dynamic
- [x] Fix font weights
- [x] Update border color
- [x] Add top margin to button
- [x] Fix behavior when window is resized

### CODE REVIEW

File: static/js/redux/ui/modals/PeerModal.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ const ghostCard = (
           style={{ backgroundImage: "url(/static/img/blank.jpg)" }}
         />
         <div className="user-info">
-          <div className="ghost-name">H</div>
+          <div className="ghost-name" />
```
Comment (JJamesWWang) [inline static/js/redux/ui/modals/PeerModal.tsx:55]: Not the H 😭

File: static/js/redux/ui/modals/PeerModal.tsx [hunk 2]
```diff
@@ -234,13 +234,19 @@ const PeerModal = () => {
         <h4>Your Classmates</h4>
         <div className="key">
           <div className="key-entry">
-            <div className="course-color-circle" style={{ backgroundColor: "#ddd" }}>
+            <div
+              className="course-color-circle"
+              style={{ backgroundColor: "rgb(128, 128, 128)" }}
```
Comment (JJamesWWang) [inline static/js/redux/ui/modals/PeerModal.tsx]: If it's better you can extract this into a variable too

**Comment by JJamesWWang** [comment]
image

Can you change the border color on the left and add some margin above the button

**Comment by ruibrii** [comment]
Fixed the margin and border color. Still have to look into behavior when browser window is resized.

**Review by JJamesWWang — APPROVED** [review]
Looks good

**Pull request merged**